components/layout_2020/flow/mod.rs

```diff
@@ -1504,3 +1515,10 @@ impl PlacementState {
         )
     }
 }
+
+fn clamp_au(value: Au, min: Au, max: Option<Au>) -> Au {
+    match max {
+        Some(max_value) => value.min(max_value).max(min),
+        None => value.max(min),
+    }
+}
```

Comment:
Can you add this as a trait on `Au` to `style_ext.rs` and also call it `fn clamp_between_extremums(...)`. I think that's a slightly cleaner change as it will prevent code from having to change all over.

Reply:
thanks for the review. done in [523e7be](https://github.com/servo/servo/pull/31083/commits/523e7bef67e5e182746a471f524f9fff93e3c0ab)